membership-baseline.md

```diff
@@ -0,0 +1,169 @@
+# Overview:
+The null Community operates on a decentralized model where participants play essential roles as volunteers, core team members, and board members. While the community does have a membership program, it's important to note that the majority of our resources and activities are accessible to both members and non-members alike.
+
+## Membership
+The null Community offers a membership program that provides additional benefits and opportunities for active participants. Members have the privilege of serving as core team members, volunteers, and potentially as board members. However, it's worth emphasizing that membership is not a prerequisite for engaging in most of the community's activities.
```

Comment:
Not everyone gets the free membership in current situation also
https://owasp.org/www-policy/legal/bylaws#:~:text=SECTION%204.02%20Qualifications

also this is highly debate topic in OWASP chapters also so this might be something for us to figure out on our own.

Reply:
I have modified the statement, I think we can live with this for now "Members have the privilege of serving as core team members, and potentially as board members. However, it's worth emphasizing that membership is not a prerequisite for engaging in most of the community's activities."